src/crepe/facade/SDLContext.h

```diff
@@ -221,6 +231,9 @@ class SDLContext {
 
 	//! renderer for the crepe engine
 	std::unique_ptr<SDL_Renderer, std::function<void(SDL_Renderer *)>> game_renderer;
+
+	//! black bars rectangle to draw
+	SDL_FRect black_bars[2];
```

Comment:
Deze is niet geïnitialiseerd, C structs kunnen zichzelf niet initialiseren net als C++ structs dus hier moet nog een `= {}` achter